people.js

```diff
@@ -0,0 +1,100 @@
+/* jshint esversion: 6 */
+
+const jeremy = {
+  name: 'Jeremy',
+  favCuisine: ['Lebanese', 'Indian', 'French', 'Japanese'],
```

Comment:
I would create const for favCuisine instead of raw-strings

Reply:
we will need in this case an entity cuisine type ?

Reply:
Will do in a next PR

Reply:
Yes, I would say we need the cuisine entity as well. So we have the IDs, and we don't care about the name itself for the relations ;)

Reply:
Either you create the type Cuisine(id,name) or you create some const for the Cuisine name... up to you.